docs/source/releases/104_0.rst

```diff
@@ -0,0 +1,84 @@
+========================
+104.0 - Restore Point
+========================
+
+
+The Avocado team is proud to present another release: Avocado 104.0,
+AKA "Restore Point", is now available!
+
+
+Release documentation: `Avocado 104.0
+<http://avocado-framework.readthedocs.io/en/104.0/>`_
+
+
+Users/Test Writers
+==================
+
+
+* The minimal Python version requirement now is 3.8. Python 3.7 and
+  earlier are not tested nor supported starting with this release.
+* The result.json test attributes related to time has been renamed in version 104.0
+  to correspond to `job.result.tests` in Job API. The Difference between new and old::
+
+        time_start = start
+        actual_time_start = actual_start
+        time_end = end
+        actual_time_end = actual_end
+        time_elapsed = time
+
+* The parsing of avocado configuration files has been improved to
+  show better error messages during parsing error.
+* Unused and legacy ``simpletest.*`` settings have been removed
+  from avocado as leftover of legacy runner.
+
+
+Utility Modules
+===============
+
+
+* :func:`avocado/utils/network/interfaces.get_device_IPI_name`, got fix for
```

Comment:
This reference is wrong, should be: `avocado.utils.network.interfaces.NetworkInterface.get_device_IPI_name`